docs/api.md

```diff
@@ -83,6 +84,22 @@ attributes key like `<tag attributes={{'data-custom-attr': 'foo'}} />`.
 
 - - -
 
+### <a id="JSX"></a> `JSX`
+
+A JSX plugin provide the transformation of the open standard JSX with the use of Babel.
+Firstly install this [babel-plugin-transform-jsx](https://www.npmjs.com/package/babel-plugin-transform-jsx) plugin,
+in your babel configuration use the following:
+```
+{
+ "plugins": [["babel-plugin-transform-jsx", { "module": "@cycle/dom/jsx" }]]
+}
+```
+Note there is now no need to add `\/\* @jsx *\/\` at the top of the files using JSX.
+
+For more information read the documentations [here]((https://www.npmjs.com/package/babel-plugin-transform-jsx)).
+
+- - -
+
```

Comment:
No it's inside `src/cycle-dom.js` but when I put it into `jsx.js` I had to adjust the `make-api-docs.js` to add `jsx.js` into an array but the output from running the script was a whole new section with header titles and links, I'll investigate if that file can just append to the current titles etc.